src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
1. It's redundant to create three instances of Lottery object. You can create it only once: 
 Lottery lottery = new Lottery();
2. There is incorrect naming variable
3. To create several balls you can use a loop and print them in the console: 
for (int i = 0; i < 3; i++) { 
System.out.println(lottery.getRandomBall()); 
} 
Also, you can use constant variable to avoid using of magic number = 3 like 
RANDOM_BALL_COUNT = 3;